src/control/lib/daos/pool_property.go

```diff
@@ -340,6 +340,61 @@ func PoolProperties() PoolPropertyMap {
 				},
 			},
 		},
+		"checkpoint": {
+			Property: PoolProperty{
+				Number:      PoolPropertyCheckpointMode,
+				Description: "WAL Checkpointing behavior",
+			},
+			values: map[string]uint64{
+				"disabled": PoolCheckpointDisabled,
+				"timed":    PoolCheckpointTimed,
+				"lazy":     PoolCheckpointLazy,
+			},
+		},
+		"checkpoint_freq": {
+			Property: PoolProperty{
+				Number:      PoolPropertyCheckpointFreq,
+				Description: "WAL Checkpointing frequency",
```

Comment:
Might want to note what the unit is here, e.g. "WAL Checkpointing frequency, in seconds".